lib/components/List/ListItem.tsx

```diff
@@ -27,6 +28,10 @@ export interface ListItemProps extends ListItemBaseProps, ListItemHeaderProps {
   label?: ReactNode | (() => ReactElement);
   /** Optional badge */
   badge?: BadgeProps | ReactNode | undefined;
+  /** Whether to display the item with a link icon */
+  linkIcon?: boolean;
+  /** onMouseEnter event handler */
+  onMouseEnter?: HTMLProps<HTMLLIElement>['onMouseEnter'];
```

Comment:
Maybe we should move `onMouseEnter` to ListItemBase?

Reply:
Good point! Moved it 😊